app/web_ui/src/routes/(app)/docs/rag_configs/[project_id]/[rag_config_id]/rag_config/+page.svelte

```diff
@@ -0,0 +1,207 @@
+<script lang="ts">
+  import { page } from "$app/stores"
+  import { client } from "$lib/api_client"
+  import type { ChunkerType, RagConfigWithSubConfigs } from "$lib/types"
+  import { createKilnError, type KilnError } from "$lib/utils/error_handlers"
+  import AppPage from "../../../../../app_page.svelte"
+  import PropertyList from "$lib/ui/property_list.svelte"
+  import { onMount } from "svelte"
+  import {
+    chunker_type_format,
+    extractor_output_format,
+    formatDate,
+  } from "$lib/utils/formatters"
+  import {
+    embedding_model_name,
+    load_available_embedding_models,
+    load_available_models,
+    model_name,
+    provider_name_from_id,
+  } from "$lib/stores"
+
+  $: project_id = $page.params.project_id
+  $: rag_config_id = $page.params.rag_config_id
+
+  let loading: boolean = false
+  let error: KilnError | null = null
+  let rag_config: RagConfigWithSubConfigs | null = null
+
+  onMount(async () => {
+    // need to load available models to get the model store populated
+    await load_available_models()
+    await load_available_embedding_models()
+
+    await get_rag_config()
+  })
+
+  async function get_rag_config() {
+    try {
+      loading = true
+      const { error: get_rag_config_error, data: rag_config_data } =
+        await client.GET(
+          "/api/projects/{project_id}/rag_configs/{rag_config_id}",
+          {
+            params: {
+              path: {
+                project_id,
+                rag_config_id,
+              },
+            },
+          },
+        )
+
+      if (get_rag_config_error) {
+        error = createKilnError(get_rag_config_error)
+        return
+      }
+
+      rag_config = rag_config_data
+    } finally {
+      loading = false
+    }
+  }
+
+  function tooltip_for_chunker_type(chunker_type: ChunkerType): string {
+    const friendly_chunker_type = chunker_type_format(chunker_type)
+    switch (chunker_type) {
+      case "fixed_window":
+        return `The ${friendly_chunker_type} chunking algorithm splits the text into fixed-size chunks of a specified number of words, while respecting sentence boundaries.`
+      default: {
+        // trigger a type error if there is a new output format, but don't handle it
+        // in the switch
+        const exhaustiveCheck: never = chunker_type
+        return exhaustiveCheck
+      }
+    }
+  }
+</script>
+
+<div class="max-w-[1400px]">
+  <AppPage
+    title="Search Tool (RAG)"
+    subtitle={loading ? "" : `Name: ${rag_config?.name}` || "Unknown"}
```

Comment:
|| "Unknown" will never trigger, because `Name: ${rag_config?.name}` will never be none. That said, we wouldn't want to render "Unknown", leave that to the error message.

Maybe 
```
rag_config?.name ? `Name: ${rag_config?.name}` : undefined`
```